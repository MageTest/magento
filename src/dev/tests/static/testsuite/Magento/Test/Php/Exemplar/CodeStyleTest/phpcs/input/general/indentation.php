<?php
if (true) {
    $spaces = 4;
	$tab = 1;
$spaces = 0;
 $spaces = 1;
  $spaces = 2;
   $spaces = 3;
    $spaces = 4;
    if (true) {
        $spaces = 5;
      	 $spaces = 7;
    }
}
