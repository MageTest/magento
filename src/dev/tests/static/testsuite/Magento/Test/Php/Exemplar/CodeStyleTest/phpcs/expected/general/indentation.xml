<?xml version="1.0" encoding="UTF-8"?>
<config>
    <error line="4"/>
    <error line="5"/>
    <error line="6"/>
    <error line="7"/>
    <error line="8"/>
    <error line="12"/>
</config>
